test/test_wofpy_soap.py

```diff
@@ -1,4 +1,4 @@
-import suds
+from suds.client import Client
```

Comment:
ok. FYI, I'm exposed to `suds-jurko` via `ulmo` and possibly `pyoos` (I can't remember). It's been a painless drop-in replacement for the now deprecated/abandoned `suds`.